misc/python/materialize/mzcompose/services/minio.py

```diff
@@ -47,3 +47,19 @@ def __init__(
                 },
             },
         )
+
+
+class Mc(Service):
+    def __init__(
+        self,
+        name: str = "mc",
+        image: str = "minio/mc:RELEASE.2023-07-07T05-25-51Z",
+    ) -> None:
+        super().__init__(
+            name=name,
+            config={
+                "image": image,
+                "volumes": ["../../mzdata/mc:/root/.mc"],
```

Comment:
I am afraid it seems this volume mount causes the junitxml file to have the wrong permissions, hence the CI failure. I guess you use it in order to use the `alias` mc functionality?

Reply:
Any suggestions or preference on how to fix this one?

Reply:
I will give it another try today.

Reply:
A fix for the test - https://github.com/MaterializeInc/materialize/commit/61e35540ac74b3f7aba387e12df79f211428759a